### PR TITLE
MQE-2023: CHANGELOG.MD and Composer version bump

### PR DESCRIPTION
**Pull request by soumyau**
Updated changelog

### Fixed Issues (if relevant)

1. magento/magento2-functional-testing-framework#<issue_number>: Issue title
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/verification tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)
 - [ ] Changes to Framework doesn't have backward incompatible changes for tests or have related Pull Request with fixes to tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
------
+3.0.0 RC1
+---------
+
+### Enhancements
+* Customizability
+    * Introduced MFTF helpers `<helper>` to create custom actions outside of MFTF.
+    * Removed deprecated actions `<executeSelenium>` and `<performOn>`.
+* Maintainability
+    * Schema updates for test entities to only allow single entity per file.
```
Comment (jilu1) [inline CHANGELOG.md]: Exceptions to this are Data and MetaData

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
-3.0.0
------
+3.0.0 RC1
+---------
+
+### Enhancements
+* Customizability
```
Comment (jilu1) [inline CHANGELOG.md]: Please also add sub-folders are supported for Test modules.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,30 @@
...
+    * Introduced MFTF helpers `<helper>` to create custom actions outside of MFTF.
+    * Removed deprecated actions `<executeSelenium>` and `<performOn>`.
+* Maintainability
+    * Schema updates for test entities to only allow single entity per file.
+    * Removed support to read test entities from `<magento>dev/tests/acceptance/tests/functional/Magento/FunctionalTest`.
```
Comment (jilu1) [inline CHANGELOG.md]: Please also add \<module file=""/`> is removed from Suite schema

File: CHANGELOG.md (hunk 1 above)
Comment (soumyau) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (soumyau) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (soumyau) [reply CHANGELOG.md]: done

**Pull request merged**